pyproject.toml

```
@@ -13,7 +13,7 @@ classifiers = [
   "Environment :: Web Environment",
   "Framework :: Django",
   "Framework :: Django :: 4.2",
-  "Framework :: Django :: 5.0",
+  "Framework :: Django :: 5.1",
```

5.0 sigue siendo compatible.

```suggestion
  "Framework :: Django :: 5.0",
  "Framework :: Django :: 5.1",
```